geotrek/core/filters.py

```diff
@@ -77,18 +78,17 @@ def _topology_filter(self, qs, edges):
             return qs.filter(pk__in=[topo.pk for topo in overlapping])
 
 
-class PathFilterSet(ZoningFilterSet, StructureRelatedFilterSet):
-    length = OptionalRangeFilter(label=_('length'))
+class PathFilterSet(AltimetryAllGeometriesFilterSet, ZoningFilterSet, StructureRelatedFilterSet):
     name = CharFilter(label=_('Name'), lookup_expr='icontains')
     comments = CharFilter(label=_('Comments'), lookup_expr='icontains')
 
     class Meta(StructureRelatedFilterSet.Meta):
         model = Path
         fields = StructureRelatedFilterSet.Meta.fields + \
-            ['valid', 'length', 'networks', 'usages', 'comfort', 'stake', 'draft', ]
+            ['valid', 'networks', 'usages', 'comfort', 'stake', 'draft', ]
```

Comment:
sure you delete length here ?

Reply:
yes or it is duplicated with allgeometriesfilterset

Reply:
but you don't inheritate Meta from it

Reply:
nor zoning too

Reply:
https://django-filter.readthedocs.io/en/stable/guide/usage.html#the-filter

Reply:
il me semble que ceux dans meta c'est ceux qui sont autogénéré par django filter et dès que tu ajoutes un champ en definition il est pris en compte dans le filterset

Reply:
oui vu la doc. ce n'est pas standard mais tu as fait ce qu'il faut. ok